app/views/browse/deployment.html

```diff
@@ -1,142 +1,266 @@
-                      <ng-include src=" 'views/browse/_deployment-details.html' "></ng-include>
+        </div>
+      </div><!-- /middle-header-->
+      <div class="middle-content">
+        <div class="container-fluid">
+          <div class="row" ng-if="loaded">
+            <div class="col-md-12" ng-class="{ 'hide-tabs' : !deployment }">
+              <uib-tabset>
+                <uib-tab active="selectedTab.details">
+                  <uib-tab-heading>Details</uib-tab-heading>
+                  <div class="resource-details" ng-if="deployment">
+                    <div class="row">
+                      <div class="col-lg-6">
+                        <h3>Status</h3>
+                        <dl class="dl-horizontal left">
+                          <dt>Last Version:</dt>
+                          <dd>{{deployment | annotation : 'deployment.kubernetes.io/revision'}}</dd>
+                          <dt>Replicas:</dt>
+                          <dd>
+                            <replicas spec="deployment.spec.replicas"
+                                      disable-scaling="autoscalers.length"
+                                      scale-fn="scale(replicas)"
+                                      deployment="deployment"></replicas>
+                            <span ng-if="autoscalers.length">(autoscaled)</span>
+                            <div ng-if="deployment.status.updatedReplicas">
+                              {{deployment.status.updatedReplicas}} up to date
+                            </div>
+                            <div ng-if="deployment.status.availableReplicas || deployment.status.unavailableReplicas">
+                              <span ng-if="deployment.status.availableReplicas">{{deployment.status.availableReplicas}} available<span ng-if="deployment.status.unavailableReplicas">,</span></span>
+                              <span ng-if="deployment.status.unavailableReplicas">{{deployment.status.unavailableReplicas}} unavailable</span>
+                            </div>
+                          </dd>
+                          <dt>Paused:</dt>
+                          <dd>{{!!deployment.spec.paused}}</dd>
+                        </dl>
+                      </div>
+                      <div class="col-lg-6">
+                        <h3>Configuration</h3>
+                        <dl class="dl-horizontal left">
+                          <dt>Selectors:</dt>
+                          <dd>
+                            <selector selector="deployment.spec.selector"></selector>
+                          </dd>
+                          <dt>Strategy:</dt>
+                          <dd>{{deployment.spec.strategy.type | sentenceCase}}</dd>
+                          <dt ng-if-start="deployment.spec.strategy.rollingUpdate">Max Unavailable:</dt>
+                          <dd>
+                            {{deployment.spec.strategy.rollingUpdate.maxUnavailable}}
+                            <span data-toggle="tooltip"
```

Comment:
open to changing this, but it felt pretty weird with the colon :/ not sure what's best